install.sh

```diff
@@ -176,10 +173,19 @@ curl -k -L -o setup.sh https://raw.githubusercontent.com/uibakery/self-hosted/ma
 curl -k -L -o update.sh https://raw.githubusercontent.com/uibakery/self-hosted/main/update.sh
 curl -k -L -o docker-compose.yml https://raw.githubusercontent.com/uibakery/self-hosted/main/docker-compose.yml
 
+# comment out the curl above and uncomment this for testing and development
+#cp ../setup.sh ./setup.sh
+#cp ../update.sh ./update.sh
+
 echo "Configuring application settings....."
-sudo bash ./setup.sh
+
+export SESSION_ID
+export GET_KEY_LINK
+export LICENCE_SERVER
+./setup.sh
```

Comment:
This way we can pass variables from the main script.

Reply:
add this as a comment please